.schema/config.schema.json

```diff
@@ -574,6 +574,26 @@
             ]
           ]
         },
+        "jwk_max_wait": {
```

Comment:
To keep consistency maybe do `jwks_`? :)

Reply:
makes sense, changed to `jwks_`